### PR TITLE
publish a version on npm every time something is merged on master

### PR DESCRIPTION
**Pull request by giogonzo**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+build:
+  install:
+    image: node:6
+    commands:
+      - npm install
```
Comment (gabro) [inline .drone.yml]: You can use yarn (or maybe node:8?)

**Review by gabro — COMMENTED** [review]
Doesn't this hang forever waiting for user input without a .smoothreleaserc?

**Comment by giogonzo** [comment]
@gabro yes, I'll add one

**Comment by giogonzo** [comment]
@FrancescoCioria will this `.smooth-releaserc` be enough?

**Review by FrancescoCioria — CHANGES_REQUESTED** [review]
`smooth-release` will also try to push some changes to origin.
I remember we had to add a couple of steps to enable the CI to push
@gabro am I right? can you help with that?

**Review by FrancescoCioria — APPROVED** [review]

**Review by gabro — APPROVED** [review]

**Pull request merged**